applications/metaquity_nft_provenance_tracker.md

```diff
@@ -0,0 +1,144 @@
+
+### Overview
+
+- **Total Estimated Duration:** 2,5 months
+- **Full-Time Equivalent (FTE):**  2 FTE (full-time employees)
+- **Total Costs:** 20,000 USD
+
+### Milestone
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation |  We will provide both readme.md and official architectural documentation of the NFT scanner module |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | Dockerfiles and docker-compose.yml for running a development environment that locally spins up, which will be able to interact with the database to fetch the indexed information locally. |
+| **0e.** | Article | Add an article covering how to integrate NFT-SDK in a project. |
+| 1. | NFT Schema Development | Development of data models for the indexing of the NFT-related extrinsic so that users can get compressive information of the NFT lifecycle  |
+| 2. | Collection schema development | Development of NFT-collection schema models |
+| 3. | NFT fractionalization schema development | Development of fractionalization schema models |
+| 4. | NFT lifecycle schema development | Developing schema models for the NFT lifecycle, which includes the ownership, transfer, locking, or fractionalization of NFT |
+| 5. | NFT pallet event indexer  | Development of the module to monitor the latest block generation via subsquid |
+| 6. | Asset pallet event and fractionalization indexer  | Development of the module to monitor the pallet NFT actions and NFT fractionalization pallet actions |
+| 7. | Integration of subsquid  | Using subsquid to get the latest block information  |
+| 8. | Implementation of APIs  | Development of endpoints that can ge used for SDKs request to fetch more details about the NFT lifecycle  |
+| 9. | Implementation of SDK  | Develop SDK to facilitate the user's call to action |
+| 10. | NFT indexer module | Index all the data that is scanned by the scanner module and pass it to the database layer. |
+| 11. | Design and development of the NFT provenance SDK | A typescript-based SDK that consists of API calls to the database to fetch the NFT lifecycle. |
+| 12. | Technical Documentation for the SDK | We will provide a compressive technical document outlining the process of integrating the SDK, available functionality, and data model types of the data that is fetched.  |
```

Comment:
In general, these deliverables are very vague:

- I'm not sure we need multiple deliverables that are concerned with implementing "schema models". I'd expect that to be part of a module/library/SDK etc. Don't get me wrong - I think it's good that you're doing that (rather than using vanillajs with no custom data structures in the form of types/interfaces/classes, etc.) but I don't think there's a need to break them into deliverables. It's sufficient if you mention it in the spec of a deliverable (e.g. something like _the SDK is going to be implemented using TypeScript and we're going to use custom interfaces for X, Y and Z_).
- Not sure what's the difference between _9. | Implementation of SDK_ and _11. | Design and development of the NFT provenance SDK_
- Not sure what is meant by "facilitate the user's call to action"
- It'd be helpful to know which technologies and languages are being used in each deliverable.